ssl/rio/poll_immediate.c

```diff
@@ -91,10 +91,10 @@ static int poll_translate_ssl_quic(SSL *ssl,
 
     if (fd2 == fd1) {
         fd2 = -1;
-        fd1_w = fd1_w || fd2_w;
+        fd1_w = fd2_w;
```

Comment:
Did you confer with whoever wrote this?

Reply:
No, but I didn't change any functionality. For this specific line, `fd1_w` will always equal 0, so that's why the change is equivalent.

Reply:
I think the change is correct. Here we deal with case when application wants to wait until SSL object is ready either for writing or reading. If the SSL object is associated with the same socket (file descriptor) then we want to put the fd into poll array just one time.